pandas/tseries/tools.py

```diff
@@ -224,6 +224,8 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     infer_datetime_format : boolean, default False
         If no `format` is given, try to infer the format based on the first
         datetime string. Provides a large speed-up in many cases.
+    origin : datetime.date, default None
```

Comment:
add an example of why you would want to do this (you can add in the ``Examples`` section).